.pre-commit-config.yaml

```diff
@@ -277,7 +277,7 @@ repos:
         entry: "^\\s*from\\s+\\."
         pass_filenames: true
         files: \.py$
-        exclude: ^tests/
+        exclude: ^tests/|^chart/tests/
```

Comment:
It is needed?

Reply:
I think this one is needed as if it's not excluded it will try to parse those files

Reply:
In my opinion, this is expected behavior. Instead, we should use the correct imports.
````diff
-from .helm_template_generator import render_chart
+from tests.helm_template_generator import render_chart
````

Reply:
<img width="678" alt="Screen Shot 2020-10-19 at 5 29 24 PM" src="https://user-images.githubusercontent.com/2644098/97211212-ef693d80-177b-11eb-8b74-0cab89306949.png">
That seems to work on the tests but any idea how to get it to register in pycharm? want to make sure that's not too difficult.

Reply:
<img width="786" alt="Screen Shot 2020-10-26 at 11 10 42 AM" src="https://user-images.githubusercontent.com/2644098/97211278-0a3bb200-177c-11eb-9ea9-dad29c2d8fac.png">
That works on the docker-based tests, but is shows an error on PyCharm which might make development difficult. Any idea how to fix that? I think pycharm usually assumes root.

Reply:
1. Click **File** => **Project structure**
1. On the left menu, select "**Project settings** => **Modules**
1. Click **+** >> **Add new module** to a new project module
1. Select **Python**
1. **Next**
1. Type **Chart** in "Module name"" field
1. **Finish**
1. Click **Add Content Root** and add `/chart` directory again

Reply:
<img width="1038" alt="Screenshot 2020-10-26 at 19 33 49" src="https://user-images.githubusercontent.com/12058428/97213653-324e0400-17c2-11eb-83f9-544333fbfcb3.png">